shared/test/commonUtilTest.js

```diff
@@ -44,10 +44,10 @@ it("should return true for valid yaml string", () => {
     groups:
     - name: newyork_users
       access:
-      - userByEmail: benjentargaryen.327201@gmail.com
+      - user: benjentargaryen.327201@gmail.com
     - name: chicago_users
       access:
-      - userByEmail: melisandresnow.388027@gmail.com`)).is.true;
+      - user: melisandresnow.388027@gmail.com`)).is.true;
```

Comment:
Do we have a Google Group email that can be utilized as a test?

Reply:
This particular test is just validating if it's valid YAML, the text could say anything. We don't have any tests or integration tests for checking if permission modifications work. Still manually verifying all of that.

Reply:
ok. I'll add something to the backlog. Thanks, Chris